root/etc/e-smith/templates/etc/httpd/conf.d/tancredi.conf/10base

```diff
@@ -24,3 +24,5 @@
     ProxyPass "fcgi://127.0.0.1:9605/usr/share/tancredi/public/api-v1.php"
 </Location>
 
+XSendFile on
+XSendFilePath /var/lib/tancredi/data/firmware
```

Comment:
I verified that this global scope setting can be safely extended by other virtual hosts, if needed.

Another virtual host can send a file with X-Sendfile from the same directory, but this is a false issue, as the filesystem permissions are shared among virtualhosts on the same machine.